de.thejackimonster.KnockKnock.json

```diff
@@ -0,0 +1,59 @@
+{
+	"app-id": "de.thejackimonster.KnockKnock",
+	"runtime": "org.gnome.Platform",
```

Comment:
why does it need gnome runtime?

Reply:
GTK3 or 4? GTK 3 is in freedesktop sdk and libhandy is easy to build.

Reply:
But doesn't the GNOME runtime already ship libhandy and GTK3? Is it really better to use freedesktop and add additional dependencies manually to maintain than relying on generally available runtimes that would increase overall redundant usage of them decreasing additional space required for flatpaks on the host?

I pretty much see arguments for both ways here.